monkeys/build.gradle.kts

```diff
@@ -17,13 +17,12 @@
  */
 
 import com.wire.kalium.plugins.commonDokkaConfig
-import com.wire.kalium.plugins.commonJvmConfig
 
 @Suppress("DSL_SCOPE_VIOLATION")
 
 plugins {
     application
-    kotlin("multiplatform")
+    kotlin("jvm")
```

Comment:
Why do we need to drop Multiplatform support?

Reply:
To simplify the development. This app won't be realistically running out of the JVM